modules/Microsoft.Compute/virtualMachines/deploy.bicep

```diff
@@ -325,8 +330,9 @@ var accountSasProperties = {
   signedResourceTypes: 'o'
   signedProtocol: 'https'
 }
-
-var identityType = systemAssignedIdentity ? (!empty(userAssignedIdentities) ? 'SystemAssigned,UserAssigned' : 'SystemAssigned') : (!empty(userAssignedIdentities) ? 'UserAssigned' : 'None')
+// change SystemAssignedIdentity to True if AADJoin is enabled.
+var varsystemAssignedIdentity = extensionAadJoinConfig.enabled ? true : systemAssignedIdentity
```

Comment:
Should be 'systemAssignedIdentityVar'

Reply:
**Thanks Alexander. I'll update and resubmit.**